src/appleseed/renderer/modeling/scene/assembly.cpp

```diff
@@ -347,16 +345,6 @@ Dictionary AssemblyFactory::get_model_metadata() const
 DictionaryArray AssemblyFactory::get_input_metadata() const
 {
     DictionaryArray metadata;
-
-    metadata.push_back(
```

Comment:
We should provide a project migration step that removes this flag from assemblies.

Reply:
Done